cli/flox-types/src/catalog/mod.rs

```diff
@@ -18,7 +18,7 @@ pub use element::{Element, PublishElement};
 mod eval;
 pub use eval::Eval;
 mod source;
-pub use source::Source;
+pub use source::{Locked, Source};
```

Comment:
This locked flakeref representation was flawed earlier already.
We can use the `runix` one or keep the `Value`, `*Map` representation

Reply:
Already fixed in https://github.com/flox/flox/pull/713/commits/080fd02982829959be977a03ef8db7d98403c520